controllers/apim/ratelimitpolicy_finalizers.go

```diff
@@ -65,8 +65,11 @@ func (r *RateLimitPolicyReconciler) computeFinalizeGatewayDiff(ctx context.Conte
 		LeftGateways: nil,
 	}
 
-	rlpGwKeys, err := r.rlpGatewayKeys(ctx, rlp)
-	if err != nil {
+	var rlpGwKeys []client.ObjectKey
+	var err error
```

Comment:
I see this change quite explicit here, but I'm failing to see why it's needed to define them instead of the way it was

Reply:
me neither. 

I will fix that